custom-recipes/nlp-spell-checker-correction/recipe.json

```
@@ -0,0 +1,196 @@
+// This file is the descriptor for the Custom code recipe typo-and-lemma
+{
+    // Meta data for display purposes
+    "meta": {
+        // label: name of the recipe as displayed, should be short
+        "label": "Spell Checker",
+        // description: longer string to help end users understand what this recipe does
+        "description": "Recipe to correct misspelled words in text. <br> Language is either given or detected.",
+        // icon: must be one of the FontAwesome 3.2.1 icons, complete list here at https://fontawesome.com/v3.2.1/icons/
+        "icon": "icon-file-text-alt"
+    },
+    "selectableFromDataset": "input_dataset",
+    "kind": "PYTHON",
+    
+
+    // Inputs and outputs are defined by roles. In the recipe's I/O tab, the user can associate one
+    // or more dataset to each input and output role.
+
+    // The "arity" field indicates whether the user can associate several datasets to the role ('NARY')
+    // or at most one ('UNARY'). The "required" field indicates whether the user is allowed to
+    // associate no dataset with the role.
+
+    "inputRoles": [
+        {
+            "name": "input_dataset",
+            "label": "Input dataset",
+            "description": "The input dataset from which you want to correct misspelled words.",
```

Small nitpick, we avoid using "." if it's not a valid sentence.
```suggestion
            "description": "Dataset with text columns containing misspelled words",
```